src/tir/transforms/storage_rewrite.cc

```diff
@@ -206,6 +234,8 @@ class LinearAccessPatternFinder final : public StmtExprVisitor {
   bool in_thread_env_{false};
   // The scope stack.
   std::vector<StmtEntry> scope_;
+  // This is a map to connect TVMValues to real allocations
+  std::unordered_map<const VarNode*, std::vector<const VarNode*>> value_to_alloc_;
```

Comment:
could you update the comment to better explain the keys and values, and the rules for when something should be added here?